lib/src/main/java/io/ably/lib/types/ChannelOptions.java

```diff
@@ -5,9 +5,22 @@
 import io.ably.lib.util.Crypto.ChannelCipher;
 
 public class ChannelOptions {
-	public boolean encrypted;
+
+	/**
+	 * Cipher in use.
+	 */
+	private ChannelCipher cipher;
+
+	/**
+	 * Parameters for the cipher.
+	 */
 	public Object cipherParams;
 
+	/**
+	 * Are these options encrypted or not?
```

Comment:
"Whether or not this ChannelOptions is encrypted"

I don't like a question here.

Reply:
@paddybyers fixed.